frontend/src/metabase/visualizations/components/ChartTooltip/StackedDataTooltip/StackedDataTooltip.tsx

```diff
@@ -1,26 +1,26 @@
 import React from "react";
 import { TooltipRow, TooltipTotalRow } from "../TooltipRow";
-import { TooltipModel } from "../types";
+import { StackedTooltipModel } from "../types";
```

Comment:
I really like using `import type` to make it easier to easily read which imports are functional or not.
```suggestion
import type { StackedTooltipModel } from "../types";
```